ansible_runner/runner_config.py

```diff
@@ -129,6 +129,7 @@ def prepare(self):
             self.env['ANSIBLE_STDOUT_CALLBACK'] = 'awx_display'
         self.env['ANSIBLE_RETRY_FILES_ENABLED'] = 'False'
         self.env['ANSIBLE_HOST_KEY_CHECKING'] = 'False'
+        self.env['PYTHONUNBUFFERED'] = 'True'
```

Comment:
This didn't work either, which surprises me, but I don't confess to be a python expert.